src/Sulu/Bundle/MigrationBundle/Command/MigrationDiffCommand.php

```diff
@@ -0,0 +1,137 @@
+        );
+        $this->addOption('console-path', null, InputOption::VALUE_REQUIRED, 'Path to the bin/console script', 'bin/adminconsole');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $consolePath = $input->getOption('console-path');
+        $ui = new SymfonyStyle($input, $output);
+
+        foreach (static::$databases as $database => $databaseUrl) {
+            $ui->section('Create Database: ' . $database);
+
+            $envs = [
+                'DATABASE_URL' => $databaseUrl,
+                'SYMFONY_DOTENV_VARS' => false, // see https://symfony.com/doc/6.1/components/process.html#setting-environment-variables-for-processes
+            ];
+
+            $command = new Process([
+                \PHP_BINARY,
+                $consolePath,
+                'doctrine:database:create',
+                '--no-interaction',
+                '--if-not-exists',
+            ], null, $envs);
+
+            $command->run(function($type, $buffer) use ($ui) {
+                $ui->write($buffer);
+            });
+
+            if ($exitCode = $command->getExitCode()) {
+                return $exitCode;
+            }
+
+            $ui->section('Run Migrations: ' . $database);
+
+            $command = new Process([
+                \PHP_BINARY,
+                $consolePath,
+                'doctrine:migrations:migrate',
+                '--no-interaction',
+                '--allow-no-migration',
+            ], null, $envs);
+
+            $command->run(function($type, $buffer) use ($ui) {
+                $ui->write($buffer);
+            });
+
+            if ($exitCode = $command->getExitCode()) {
+                return $exitCode;
+            }
+
+            $ui->section('Create new Migration: ' . $database);
+
+            $command = new Process([
+                \PHP_BINARY,
+                $consolePath,
+                'doctrine:migrations:diff',
+                '--no-interaction',
+                '--allow-empty-diff',
+                '--check-database-platform',
+                'true',
+                '--namespace',
+                'Sulu\Bundle\MigrationBundle\Migrations',
+            ], null, $envs);
+
+            $command->run(function($type, $buffer) use ($ui) {
+                $ui->write($buffer);
+            });
+
+            if ($exitCode = $command->getExitCode()) {
+                return $exitCode;
+            }
+
+            $this->replaceAbortWithSkip();
+        }
+
+        return 0;
+    }
+
+    private function replaceAbortWithSkip(): void
+    {
+        $finder = new Finder();
+        $finder->files()->in(\dirname(__DIR__) . '/Migrations')->name('Version*.php');
+
+        foreach ($finder as $file) {
+            \file_put_contents(
+                $file->getPathname(),
+                \str_replace('abortIf', 'skipIf', $file->getContents())
+            );
+        }
+    }
```

Comment:
This should not be required as we should make sure that `PostgreSQL`, `MySQL` and `MariaDB` migrations life in an own directory. And only the directory which is relevant is really loaded for the migrations.